CityPuzzle/Game Room/Join GameRoom/SeeEnteredRooms.xaml.cs

```diff
@@ -76,17 +78,28 @@ public static List<Room> GetData()
         void Sign_Click(object sender, EventArgs e)
         {
             string gamePin = GamePin.Text;
-            Navigation.PushAsync(new EntryGameRoomPage(gamePin));
-
+            Room current= EnteredRooms.SingleOrDefault(x => x.ID.Equals(gamePin));
+            if (current == null) Navigation.PushAsync(new EntryGameRoomPage(gamePin));
+            else RoomExistError(current);
         }
         async void Handle_ItemTapped(object sender, ItemTappedEventArgs e)
         {
-            if (e.Item == null)
+            if (e.Item != null)
             {
-                return;
+                SelectMsg(EnteredRooms[e.ItemIndex]);
             }
-    ((ListView)sender).SelectedItem = null;
+            MyListView.SelectedItem = null;
             Console.WriteLine(" " + e.Item);
         }
+        async void SelectMsg(Room selectedRoom)//exeotionus galima panaudoti
+        {
+            bool answer = await DisplayAlert("Demesio", "Ar norite testi zaidima- " + selectedRoom.ID, "Taip", "Ne");
+            if (answer == true) Console.WriteLine("Iveinu  {0}");
+        }
+        async void RoomExistError(Room selectedRoom)//exeotionus galima panaudoti
+        {
+            bool answer = await DisplayAlert("Demesio", "Jus jau registruotas šiame zaidyme. Ar norite testi zaidima- " + selectedRoom.ID, "Taip", "Ne");
+            if (answer == true) Console.WriteLine("Iveinu  {0}");
```

Comment:
Console.WriteLine is unclear and requires an argument specified in the format which is not given after it

Reply:
That was done for testing only.